cmd/history.go

```diff
@@ -72,20 +81,20 @@ func ExecuteHistoryCleanupCommand(args []string) error {
 	}
 	var matchingTags = cleanup.GetMatchingTags(&gitCandidates, &imageStreamTags, matchOption)
 
-	activeImageStreamTags, err := openshift.GetActiveImageStreamTags(namespace, image, imageStreamTags)
+	activeImageStreamTags, err := openshift.GetActiveImageStreamTags(namespace, image, matchingTags)
 	if err != nil {
 		return fmt.Errorf("could not retrieve active image stream tags for '%s/%s': %w", namespace, image, err)
 	}
 
 	inactiveTags := cleanup.GetInactiveImageTags(&activeImageStreamTags, &matchingTags)
 	inactiveTags = cleanup.LimitTags(&inactiveTags, c.Keep)
-
-	PrintImageTags(inactiveTags)
-
-	if config.Force {
-		DeleteImages(inactiveTags, image, namespace)
-	} else {
-		log.Info("--force was not specified. Nothing has been deleted.")
+	if len(inactiveTags) == 0 {
+		log.WithField("namespace", namespace).
+			WithField("imageName", image).
+			Info("No inactive image stream tags found")
```

Comment:
Same here, would opt for using `log.WithFields()`, also solves the formatting issue where the Method names are not truly aligned. It's nitpicky though ;)